Dockerfile.local

```diff
@@ -1,21 +1,9 @@
-FROM golang:alpine as builder
-MAINTAINER Jack Murdock <jack_murdock@comcast.com>
-
-# build the binary
-WORKDIR /go/src
-COPY src/ /go/src/
-
-RUN go build -o tr1d1um_linux_amd64 tr1d1um
-
-EXPOSE 6100 6101 6102
-RUN mkdir -p /etc/tr1d1um
-VOLUME /etc/tr1d1um
-
 # the actual image
 FROM alpine:latest
 RUN apk --no-cache add ca-certificates
+EXPOSE 6100 6101 6102
 RUN mkdir -p /etc/tr1d1um
 VOLUME /etc/tr1d1um
 WORKDIR /root/
-COPY --from=builder /go/src/tr1d1um_linux_amd64 .
+# we assume tr1d1um_linux_amd64 was already created before invoking this
```

Comment:
We still need to copy the image over. 

Best practices is to build the binary in one container then copy it over.